[2024-05-07 20:19:01,626] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: research_scraper.research_scraper manual__2024-05-07T20:18:58.722288+00:00 [queued]>
[2024-05-07 20:19:01,634] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: research_scraper.research_scraper manual__2024-05-07T20:18:58.722288+00:00 [queued]>
[2024-05-07 20:19:01,635] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 20:19:01,635] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-05-07 20:19:01,635] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 20:19:01,648] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): research_scraper> on 2024-05-07 20:18:58.722288+00:00
[2024-05-07 20:19:01,657] {standard_task_runner.py:52} INFO - Started process 1635 to run task
[2024-05-07 20:19:01,661] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'research_scraper', 'research_scraper', 'manual__2024-05-07T20:18:58.722288+00:00', '--job-id', '206', '--raw', '--subdir', 'DAGS_FOLDER/ResearchScraper.py', '--cfg-path', '/tmp/tmpw4m6gjqs', '--error-file', '/tmp/tmp2r856ro1']
[2024-05-07 20:19:01,662] {standard_task_runner.py:80} INFO - Job 206: Subtask research_scraper
[2024-05-07 20:19:01,719] {task_command.py:369} INFO - Running <TaskInstance: research_scraper.research_scraper manual__2024-05-07T20:18:58.722288+00:00 [running]> on host 5881f5798927
[2024-05-07 20:19:01,799] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=research_scraper
AIRFLOW_CTX_TASK_ID=research_scraper
AIRFLOW_CTX_EXECUTION_DATE=2024-05-07T20:18:58.722288+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-07T20:18:58.722288+00:00
[2024-05-07 20:19:01,804] {conn.py:381} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.228.5', 9092)]>: connecting to kafka:9092 [('192.168.228.5', 9092) IPv4]
[2024-05-07 20:19:01,804] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-05-07 20:19:01,806] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.228.5', 9092)]>: Connection complete.
[2024-05-07 20:19:01,917] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-05-07 20:19:01,917] {conn.py:1269} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-05-07 20:19:01,920] {logging_mixin.py:115} INFO - ['2']
[2024-05-07 20:19:01,920] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ResearchScraper.py", line 30, in call_scopus
    get_scopus_data()
  File "/opt/airflow/dags/scopus.py", line 64, in get_scopus_data
    start_pointer = read_start_pointer()
  File "/opt/airflow/dags/scopus.py", line 36, in read_start_pointer
    result[f'{row[0].strip()}{row[1].strip()}']= int(row[2].strip())
IndexError: list index out of range
[2024-05-07 20:19:01,933] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=research_scraper, task_id=research_scraper, execution_date=20240507T201858, start_date=20240507T201901, end_date=20240507T201901
[2024-05-07 20:19:01,942] {standard_task_runner.py:97} ERROR - Failed to execute job 206 for task research_scraper (list index out of range; 1635)
[2024-05-07 20:19:01,976] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-05-07 20:19:02,007] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
